atproto/auth/oauth/types.go

```diff
@@ -407,3 +407,23 @@ type TokenResponse struct {
 	// Refresh token, for doing additional token requests to the auth server.
 	RefreshToken string `json:"refresh_token"`
 }
+
+// Returned by [ClientApp.ProcessCallback] if the AS signals an error in the redirect URL parameters, per rfc6749 section 4.1.2.1
+//
+// NOTE: This is untrusted data and should not be e.g. rendered to HTML without appropriate escaping
+type ErrCallback struct {
```

Comment:
I think the idiom is to use `ErrSomething` when doing simple:

    var ErrSomething = errors.New("some thing")

but if defining a type (struct with `Error()` method), it should be `SomethingError`.

I'd call this something like `AuthRequestCallbackError`